Description: Disable doclint by default
 OpenJDK 8 adds and enables doclint by default. This catches issues in
 javadoc comments. It is too strict, breaks javadoc compilation and, in
 general, breaks the build for old code known to build with previous
 versions of OpenJDK.
 .
 See: http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html
 See: https://lists.fedoraproject.org/pipermail/java-devel/2014-February/005150.html
Author: Andrew John Hughes <ahughes@redhat.com>
Forwarded: not-needed
Index: b/langtools/src/jdk.javadoc/share/classes/com/sun/tools/javadoc/DocEnv.java
===================================================================
--- a/langtools/src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/WorkArounds.java
+++ b/langtools/src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/WorkArounds.java
@@ -125,7 +125,7 @@ public class WorkArounds {
         }

         if (!msgOptionSeen) {
-            doclintOpts.add(DocLint.XMSGS_OPTION);
+            return;
         }

         String sep = "";
